CHANGELOG

```diff
@@ -8,7 +8,7 @@ PySCF 2.3.0 (2023-07-04)
   - TDDFT gradients for triplet states
   - Support complex-valued h1e in fci_slow.absorb_h1e
 * Improved
-  - Update B3LYP functional to make it behave the same to ORCA
+  - Update B3LYP functional to make it behave the same to Gaussian
```

Comment:
This is misleading. B3LYP is B3LYP. The earlier implementation did not match it.

Reply:
So what's your suggestion about this changelog?

Reply:
Fix the definition of B3LYP to correspond to the original functional.